eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -104,136 +109,140 @@ stages:
     ### Additional jobs for lite/full builds ###
     - ${{ if in(parameters.scope, 'lite', 'full') }}:
 
-      - template: ../jobs/vmr-build.yml
-        parameters:
-          # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-          buildName: ${{ format('{0}_Online_CurrentSourceBuiltSdk', variables.centOSStreamName) }}
-          isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-          vmrBranch: ${{ variables.VmrBranch }}
-          architecture: x64
-          pool: ${{ parameters.pool_Linux }}
-          container: ${{ variables.centOSStreamContainer }}
-          buildFromArchive: false            # 🚫
-          buildSourceOnly: true              # ✅
-          enablePoison: false                # 🚫
-          excludeOmniSharpTests: true        # ✅
-          runOnline: true                    # ✅
-          useMonoRuntime: false              # 🚫
-          withPreviousSDK: false             # 🚫
-          reuseBuildArtifactsFrom:
-          - ${{ format('{0}_Online_MsftSdk_x64', variables.centOSStreamName) }}
-
-      - template: ../jobs/vmr-build.yml
-        parameters:
-          # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-          buildName: ${{ format('{0}_Offline_PreviousSourceBuiltSdk', variables.alpinePreviousName) }}
-          isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-          vmrBranch: ${{ variables.VmrBranch }}
-          architecture: x64
-          artifactsRid: ${{ variables.alpinePreviousX64Rid }}
-          pool: ${{ parameters.pool_Linux }}
-          container: ${{ variables.alpinePreviousContainer }}
-          targetRid: ${{ variables.alpinePreviousX64Rid }}
-          buildFromArchive: false            # 🚫
-          buildSourceOnly: true              # ✅
-          enablePoison: true                 # ✅
-          excludeOmniSharpTests: true        # ✅
-          runOnline: false                   # 🚫
-          useMonoRuntime: false              # 🚫
-          withPreviousSDK: true              # ✅
-
-      ### Additional jobs for full build ###
-      - ${{ if in(parameters.scope, 'full') }}:
-
-        # This AlmaLinux leg is intended to build with the min supported glibc version
+      - ${{ if parameters.enableStage2Builds }}:
```

Comment:
I don't consider these to be stage 2 builds.  These are stage 1 builds with a non-MSFT SDK.  

Long term I don't see value in having a variable/parameter to disable non-MSFT sdk builds.  Perhaps just commenting them out is appropriate for now.

Reply:
Should variable/parameter be used to condition `CurrentSourceBuiltSdk` builds or should all non-MSFT builds be just commented out?

We would need to disable all non-MSFT builds as the very next re-bootstrapping would cause failures in `PreviousSourceBuiltSdk` builds.

Reply:
I lean towards commenting all non-msft builds for now with a comment.  We likely will never use the variable/parameter again therefore it is adding unnecessary complexity IMO.

Reply:
OK - makes sense - will update this PR.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/43439/commits/935070d61109cb0d6d62df7cd0346418eb25b8ce